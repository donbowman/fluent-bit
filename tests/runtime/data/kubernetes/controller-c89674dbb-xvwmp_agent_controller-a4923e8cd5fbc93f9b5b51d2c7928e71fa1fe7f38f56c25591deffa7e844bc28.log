{"log":"{\"source\":\"audit.rule_table.controller\", \"time\" : \"2018-10-24 21:02:01,521\", \"state\":\"wombat\", \"rule\":{\"namespace\": \"agent\", \"name\":\"deny-bad-ips\", \"source-ip\":\"146.88.48.66\", \"dest-ip\":\"*\", \"source-port\":\"*\",\"dest-port\":\"*\", \"proto\":\"*\", \"priority\":10, \"audit\":true, \"action\":\"drop\"} }\n","stream":"stderr","time":"2018-10-24T21:02:01.521564232Z"}
