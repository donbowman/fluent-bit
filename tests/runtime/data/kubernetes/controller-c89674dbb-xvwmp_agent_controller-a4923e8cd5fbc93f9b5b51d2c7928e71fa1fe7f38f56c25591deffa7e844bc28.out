[1540414921.521564, {"log":"{\"source\":\"audit.rule_table.controller\", \"time\" : \"2018-10-24 21:02:01,521\", \"state\":\"wombat\", \"rule\":{\"namespace\": \"agent\", \"name\":\"deny-bad-ips\", \"source-ip\":\"146.88.48.66\", \"dest-ip\":\"*\", \"source-port\":\"*\",\"dest-port\":\"*\", \"proto\":\"*\", \"priority\":10, \"audit\":true, \"action\":\"drop\"} }\n", "stream":"stderr", "time":"2018-10-24T21:02:01.521564232Z", "source":"audit.rule_table.controller", "time":"2018-10-24 21:02:01.521", "state":"wombat", "rule":{"namespace":"agent", "name":"deny-bad-ips", "source-ip":"146.88.48.66", "dest-ip":"*", "source-port":"*", "dest-port":"*", "proto":"*", "priority":10, "audit":true, "action":"drop"}, "kubernetes":{"pod_name":"controller-c89674dbb-xvwmp", "namespace_name":"agent", "pod_id":"04fe728e-d7d0-11e8-9641-0a58ac1f0a6f", "labels":{"k8s-app":"agilicus-controller", "pod-template-hash":"745230866"}, "host":"aks-nodepool1-19254313-3", "container_name":"controller", "docker_id":"a4923e8cd5fbc93f9b5b51d2c7928e71fa1fe7f38f56c25591deffa7e844bc28"}}]
